### ISSUE TITLE
Pause segmentation reset contour

### ISSUE THREAD

**Issue by CornichonCosmique** [CloudCompare/CloudCompare#1633]
**Describe the bug**
Pause segmentation reset contour

**How to reproduce**
Launch segmentation
Start contouring
Pause using space to move the model (can't use the pause button by clicking on it, it will add a point behind the UI)
Ongoing contour line is erased

**Expected behaviour**
Start contouring
Pause using space
Move the model
Unpause using space
Continue contouring

**Your environment**
 - CC Version: v2.12.0 (Kyiv)
 - OS & Version: Windows 10
 - Graphics card: NVIDIA GeForce RTX 3080 Laptop GPU

**Comment by dgirardeau** [comment]
Yes, this is the normal behavior currently. The only thing hat could be made clearer is that the icons should be grayed out as long as the user hasn't 'closed' the polyline (with right click). I'll fix it in the next 2.12.1 version.

What you can do is save the polyline and restore it after you've finished moving the cloud.

What you suggest is a new behavior, and should be considered a (potential) future improvement.

**Comment by dgirardeau** [comment]
The 2.12.1 version has just been released.

**Comment by CornichonCosmique** [comment]
Merci beaucoup!
